internal/alerting/alerter.go

```diff
@@ -22,85 +14,27 @@ func (mn *MockAlerter) AlertError(e error) error {
 	return nil
 }
 
-func NewAlerter(cfg service.ErrorAlerting) (Alerter, error) {
+func NewAlerters(cfg service.ErrorAlerting) ([]Alerter, error) {
```

Comment:
In the past I've created a `type Alerters []Alerter` that can have `AlertError(e error)` defined on it. That can check for nil/empty cases so the calling code doesn't have to.

Reply:
There's an `Alerter` interface already that defines an `AlertError(e error)` method.  This slice of `Alerter`s can have a `PagerDuty` alerter and a `Slack` alerter, both of which have the `AlertError(e error)` method and satisfy that interface

Reply:
I like Alerters that wraps the `for` loop so that aggregate.go can just call `.Error(err)` and not have to worry about the implementation of Alerter(s).